protected/humhub/modules/tour/models/TourConfig.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2015 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\tour\models;
+
+use humhub\modules\space\models\Membership;
+use humhub\modules\space\models\Space;
+use humhub\modules\tour\Module;
+use Yii;
+
+/**
+ * Configurations for the introduction tour
+ * Loads and manage the configuration files
+ * Default files are in @tour/config
+ *
+ * @since 1.18
+ */
+class TourConfig
```

Comment:
I would place the class directly in the namespace `humhub\modules\tour\TourConfig`, as it does not extend any model.